app/helpers/application_helper.rb

```diff
@@ -8,6 +8,14 @@ def app_name
     #NUCore.app_name
   end
 
+  def app_name_link_text
```

Comment:
@nutodd: I'm inclined to go with @jhanggi's suggestion to use "NU Core" throughout (it would certainly simplify the code) but I'd like to get your input before going ahead.